### PR TITLE
Add delete subscription functionality

### PR DESCRIPTION
**Pull request by iWader**

### CODE REVIEW

File: src/UnsubscribeBuilder.php [hunk 1]
```diff
@@ -23,6 +23,11 @@ public function __construct(ApiClient $client, $subscriberId, $tenant, $events =
         $this->events = $events;
     }
 
+    public function delete($subscriptionId)
+    {
+        return $this->client->send('DELETE', '/subscribers/'.$this->subscriberId.'/subscriptions/' . $subscriptionId);
```
Comment (danharper) [inline src/UnsubscribeBuilder.php]:
```suggestion
        return $this->client->send('DELETE', '/subscribers/'.$this->subscriberId.'/subscriptions/'.$subscriptionId);
```

nit

**Review by danharper — APPROVED** [review]
can i… still accept here? 😂

File: src/UnsubscribeBuilder.php (hunk 1 above)
Comment (rknightuk) [reply src/UnsubscribeBuilder.php]: 👋️

Comment (iWader) [reply src/UnsubscribeBuilder.php]: download

**Pull request merged**